ruby/bitemporality.gemspec

```diff
@@ -0,0 +1,43 @@
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "bitemporal/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "bitemporality"
+  spec.version       = Bitemporality::VERSION
+  spec.authors       = ['Wenley Tong', "Brian Buchalter"]
+  spec.email         = ['wenley.tong@gusto.com', "brian.buchalter@gusto.com"]
+
+  spec.summary       = 'Adds bitemporal modeling to Active Record models'
+  # spec.description   = 'TODO'
+  spec.homepage      = "https://github.com/wenley/bitemporality"
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if spec.respond_to?(:metadata)
+    # spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
+
+    spec.metadata["homepage_uri"] = spec.homepage
+    spec.metadata["source_code_uri"] = "https://github.com/wenley/bitemporality"
+    spec.metadata["changelog_uri"] = "https://github.com/wenley/bitemporality"
+  else
+    raise "RubyGems 2.0 or newer is required to protect against " \
+      "public gem pushes."
+  end
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"
+  ]
+  spec.add_dependency "activerecord", "~> 5.0.0"
+
+  spec.add_development_dependency "bundler", "~> 1.17"
+  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "sqlite3", '~> 1.3.13'
```

Comment:
Confirming: Are these the lines that generated the `Gemfile.lock` changes?

Reply:
Yes.